From c08ecfda4c7faf64b72ce8bca7906223e02621f6 Mon Sep 17 00:00:00 2001
From: Tong Tiangen <tongtiangen@huawei.com>
Date: Thu, 9 Jun 2022 08:49:57 +0000
Subject: [PATCH 3557/4585] [Huawei] Revert "arm64: config: enable
 CONFIG_ARM64_UCE_KERNEL_RECOVERY"

Offering: HULK
hulk inclusion
category: feature
bugzilla: 187273

--------------------------------

This reverts commit 93e6b815435fa2b743447b895c7001862fdde1eb.

Signed-off-by: Tong Tiangen <tongtiangen@huawei.com>
---
 arch/arm64/configs/openeuler_defconfig | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm64/configs/openeuler_defconfig b/arch/arm64/configs/openeuler_defconfig
index eb80a85f4c3c..03d218859093 100644
--- a/arch/arm64/configs/openeuler_defconfig
+++ b/arch/arm64/configs/openeuler_defconfig
@@ -467,7 +467,6 @@ CONFIG_ARM64_UAO=y
 CONFIG_ARM64_PMEM=y
 CONFIG_ARM64_RAS_EXTN=y
 CONFIG_ARM64_CNP=y
-CONFIG_ARM64_UCE_KERNEL_RECOVERY=y
 # end of ARMv8.2 architectural features
 
 #
-- 
2.17.1

